src/db/collections/milestone.rs

```diff
@@ -247,34 +247,31 @@ impl MilestoneCollection {
         .await
     }
 
-    /// Find the newest milestone.
-    pub async fn get_newest_milestone(&self) -> Result<Option<MilestoneIndexTimestamp>, Error> {
-        self.find_one(
-            doc! {},
-            FindOneOptions::builder()
-                .sort(doc! { "at.milestone_index": BY_NEWEST })
-                .projection(doc! {
+    async fn get_milestone_by_order(&self, order: i32) -> Result<Option<MilestoneIndexTimestamp>, Error> {
```

Comment:
Maybe call this `get_milestone_sorted`?

Reply:
But it retrieves only the first element. So maybe `get_first_milestone_sorted`?